.github/workflows/tests.yml

```
@@ -160,6 +172,76 @@
+      with:
+        path: Playwright/node_modules
+        key: ${{ runner.os }}-Playwright-${{ hashFiles('Playwright/yarn.lock') }}
+        fail-on-cache-miss: true
+
+    - name: Restore cached node_modules for WebAPp
+      uses: actions/cache/restore@v4
+      with:
+        path: WebAPp/node_modules
+        key: ${{ runner.os }}-WebAPp-${{ hashFiles('WebAPp/yarn.lock') }}
+        fail-on-cache-miss: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)